### PR TITLE
Implemented feature SLB-F2 (Do keeper inactive)

### PR DESCRIPTION
**Pull request by Solykk**
Connect #16

### CODE REVIEW

File: src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java [hunk 1]
```diff
@@ -65,4 +70,76 @@ public RichMessage onReceiveSlashCommandKeeperAdd(@RequestParam("token") String
 
         return new RichMessage(response);
     }
+
+    @PostMapping(value = "/commands/keeper-dismiss", consumes = MediaType.APPLICATION_FORM_URLENCODED_VALUE)
+    public RichMessage onReceiveSlashCommandKeeperDismiss(@RequestParam("token") String token,
```
Comment (OlgaKulikova) [inline src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java]: Why so weird method name? I think better "dismissKeeper()"

Comment (OlgaKulikova) [reply src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java]: and other methods the same

Comment (Solykk) [reply src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java]: done

File: src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java [hunk 2]
```diff
@@ -65,4 +70,76 @@ public RichMessage onReceiveSlashCommandKeeperAdd(@RequestParam("token") String
...
+        int userCount = slackParsedCommand.getUserCount();
+
+        if (userCount > 1) {
+            throw new WrongCommandFormatException(String.format("We found %d slack names in your command: '%s' " +
+                            "You can not perform actions with several slack names.",
+                    slackParsedCommand.getUserCount(), slackParsedCommand.getText()));
+        }
+
+        if (userCount == 0) {
+            throw new WrongCommandFormatException(String.format("We didn't find slack name in your command. '%s' " +
```
Comment (OlgaKulikova) [inline src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java]: we didn't find any slack name in your command.

File: src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java [hunk 3]
```diff
@@ -65,4 +70,76 @@ public RichMessage onReceiveSlashCommandKeeperAdd(@RequestParam("token") String
...
+        return slackParsedCommand.getFirstUser();
+    }
+
+    private String receiveToDirections(SlackParsedCommand parsedCommand){
+
+        String textWithoutSlackNames = parsedCommand.getTextWithoutSlackNames();
+
+        if (textWithoutSlackNames.length() == 0){
+            throw new WrongCommandFormatException(String.format("We didn't find direction in your command: '%s' " +
+                    "You must write direction to perform action with keepers.", parsedCommand.getText()));
```
Comment (OlgaKulikova) [inline src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java]: We didn't find any direction in your command
You must write the direction to perform the action with keepers

File: src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java (hunk 2 above)
Comment (OlgaKulikova) [reply src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java]: You must write the user's slack name to perform the action with keepers

File: src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java [hunk 4]
```diff
@@ -65,4 +70,76 @@ public RichMessage onReceiveSlashCommandKeeperAdd(@RequestParam("token") String
...
+        }
+
+        if (textWithoutSlackNames.split(" ").length > 1){
+            throw new WrongCommandFormatException(String.format("We found several directions in your command: '%s' " +
+                    "You can perform action with keepers on one direction only.", parsedCommand.getTextWithoutSlackNames()));
```
Comment (OlgaKulikova) [inline src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java]: You can perform the action with keepers on one direction only

**Review by OlgaKulikova — APPROVED** [review]

**Pull request merged**